### PR TITLE
Update ansible.md

### PR DESCRIPTION
**Pull request by voriol**
##### Summary

the ansible path is not community/configuration-management/ansible, seems to be: netdata-agent-deployment/ansible-quickstart/

##### Component Name

README.md

##### Test Plan

N/A

### CODE REVIEW

**Review by joelhans — APPROVED** [review]
Thank you very much, @voriol! The path was changed in th community repo but it didn't get updated here.

**Review by joelhans — DISMISSED** [review]

File: docs/guides/deploy/ansible.md [hunk 1]
```diff
@@ -52,14 +52,14 @@ current directory, and remove the rest of the cloned repository, as it's not req
 
 ```bash
 git clone https://github.com/netdata/community.git
-mv community/configuration-management/ansible .
+mv netdata-agent-deployment/ansible-quickstart .
```
Comment (ilyam8) [inline docs/guides/deploy/ansible.md:55]:
```suggestion
mv community/netdata-agent-deployment/ansible-quickstart .
```

Comment (joelhans) [reply docs/guides/deploy/ansible.md:55]: Good catch, @ilyam8. @voriol can you make this change? Then we'll approve and merge.

**Review by OdysLam — APPROVED** [review]

File: docs/guides/deploy/ansible.md (hunk 1 above)
Comment (voriol) [reply docs/guides/deploy/ansible.md]: Thanks for pointing that out @ilyam8. I update the line in patch-1 branch.

**Review by ilyam8 — APPROVED** [review]

**Review by joelhans — APPROVED** [review]

**Pull request merged**